Changelog.md

```diff
@@ -5,6 +5,8 @@
 ### 🚨 Breaking changes
 
 ### ✨ New features and improvements
+- Remove the ... under "Proposed changes".
```

Comment:
These entries should go under "Internal changes", since they aren't user-facing changes. Also, they need to specify what file is being changed (for example, what is "Proposed changes"?). Keep in mind that someone should be able to read the entry and understand what it means without seeing the actual code change.